docs/reST/ref/font.rst

```diff
@@ -121,6 +121,8 @@ solves no longer exists, it will likely be removed in the future.
 
    .. versionadded:: 2.0.1 Accept an iterable of font names.
 
+   .. versionadded:: 2.1.3 Iterates through user & global fonts.
```

Comment:
I think this can be worded better for clarity
Something like "Finds fonts installed in user directories"
Because this function already supported globally installed fonts so that should not be mentioned in this `versionadded`

Reply:
Are the current changes okay? It's a little different than what you suggested.

Reply:
What does windows 10 have to do with anything?

Reply:
This implementation is only used on Windows 10 or 11 devices.

Reply:
I'm sure-ish this should work for older windows too, but I can't test that rn